feature/authentication/presentation/src/test/java/com/sanaa/presentation/screen/welcome/WelcomeViewModelTest.kt

```diff
@@ -2,19 +2,38 @@ package com.sanaa.presentation.screen.welcome
 
 import app.cash.turbine.test
 import com.google.common.truth.Truth.assertThat
+import io.mockk.coEvery
+import io.mockk.mockk
+import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.StandardTestDispatcher
+import kotlinx.coroutines.test.TestDispatcher
+import kotlinx.coroutines.test.advanceUntilIdle
 import kotlinx.coroutines.test.runTest
+import kotlinx.coroutines.test.setMain
 import org.junit.jupiter.api.BeforeEach
 import org.junit.jupiter.api.Test
+import usecase.CheckIfUserIsLoggedInUseCase
+import usecase.CreateGuestSessionUseCase
 
 @OptIn(ExperimentalCoroutinesApi::class)
 class WelcomeViewModelTest {
 
     private lateinit var viewModel: WelcomeViewModel
+    private val createGuestSessionUseCase: CreateGuestSessionUseCase = mockk()
+    private lateinit var testDispatcher: TestDispatcher
```

Comment:
initial the test dispatcher here 
```suggestion
    private lateinit var testDispatcher: TestDispatcher = StandardTestDispatcher()
```